subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -602,17 +602,40 @@ Gradle will behave as if you ran `gradle test` after a change to sources or test
 This means that unrelated changes (such as changes to build scripts) will not trigger a rebuild.
 In order to incorporate build logic changes, the continuous build must be restarted manually.
 
+Continuous build uses file system watching to detect changes to the inputs.
+If file system watching does not work on your system, then continuous build won't work either.
+Especially, continuous build does not work when using `--no-daemon`.
+See <<gradle_daemon.adoc#sec:daemon_watch_fs,Watching the file system>> for more information.
+
 === Terminating Continuous Build
 
 If Gradle is attached to an interactive input source, such as a terminal, the continuous build can be exited by pressing `CTRL-D` (On Microsoft Windows, it is required to also press `ENTER` or `RETURN` after `CTRL-D`).
 If Gradle is not attached to an interactive input source (e.g. is running as part of a script), the build process must be terminated (e.g. using the `kill` command or similar).
 If the build is being executed via the Tooling API, the build can be cancelled using the Tooling API's cancellation mechanism.
 
 [[continuous_build_limitations]]
-=== Limitations and quirks
+=== Limitations
+
+Under some circumstances continuous build will not detect changes to inputs.
+
+[[sec:continuous_build_missing_files]]
+==== Adding new root input directories
+
+Sometimes, adding root input directories does not trigger a build, due to the way file system watching works.
```

Comment:
What's a "adding a root input directory" mean here? I don't think that's clear at this point and we should define it first.

I think we should say "create" instead of "add". I'd say something like "creating a directory that was previously missing that is used as an input to a task."

Reply:
I am using `create` now.